nix/modules/fedimintd.nix

```diff
@@ -0,0 +1,233 @@
+{ config, lib, pkgs, ... }:
+with lib;
+
+let
+  eachFedimintd = filterAttrs (fedimintdName: cfg: cfg.enable) config.services.fedimintd;
+  fedimintdOpts = { config, lib, name, ... }: {
+
+    options = {
+
+      enable = mkEnableOption (lib.mdDoc "fedimint");
+
+      extraEnvironment = mkOption {
+        type = types.attrsOf types.str;
+        description = lib.mdDoc "Extra Environment variables to pass to the fedimintd.";
+        default = { };
+        example = { RUST_BACKTRACE = "1"; };
+      };
+
+      package = mkOption {
+        type = types.nullOr types.package;
+        default = pkgs.fedimintd or null;
```

Comment:
Should we default to the falke's own package definition for now? This reads to me as if we were using a non-existent nixpkgs version? Moreso, it's a bit confusing if the module from a specific branch/version of the repo will depend on a random other version of Fedimint.

Reply:
we should have an overlay that add `fedimintd` to pkgs